response_operations_ui/templates/create-ce-event.html

```diff
@@ -28,17 +28,25 @@ <h1 class="u-fs-xl">Create business event</h1>
 
     <h2 class="u-fs-l u-mt-l">{{event_name}}</h2>
 
-    {% if errors %}
-        <div class="panel panel--simple panel--error u-mb-l">
-          <div class="panel__body" id="save-error">
-            <h2 id='error-saving-message' class="u-fs-r--b">Error creating {{event_name}} date</h2>
-    {% endif %}
+    {%- with errors = get_flashed_messages(category_filter=["error"]) -%}
+        {%- if errors -%}
+            <div class="panel panel--simple panel--error u-mb-l">
+              <div class="panel__body" id="save-error">
+                <h2 id='error-saving-message' class="venus">Error updating {{event_name}} date </h2>
+                <div id='error-text'>
+                  {%- for message in errors -%}
+                  <p>{{ message }}</p>
```

Comment:
Don't we have a partial to do this bit?  If not, surely this isn't a bunch of `<p>` tags.  I would've expected it to be a list.  I've not looked at all the other templates, if that's what they're doing for bunches of errors then that's fine.